### PR TITLE
light/dark mode switch added

### PR DESCRIPTION
**Pull request by Alirezaaraby**
toggle dark/light mode added to docs

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -6,8 +6,19 @@ theme:
   name: material
   custom_dir: docs/overrides
   palette:
-    primary: black
-    accent: teal
+    - media: "(prefers-color-scheme: light)"
+      primary: black
+      accent: teal
+      toggle:
+        icon: material/brightness-7
```
Comment (atmartinez) [inline mkdocs.yml:13]: I would consider matching these icons to the FastApi [documentation](https://github.com/tiangolo/fastapi/blob/6879082b3668edd213d035b6e9a90a4bccf32e01/docs/en/mkdocs.yml#L13) to maintain consistency between the projects.

Comment (Alirezaaraby) [reply mkdocs.yml:13]: good idea

**Review by atmartinez — APPROVED** [review]
This looks good to me for this specific doc style change. I would consider swapping the accent colors to amber to match FastApi docs, but the scope of this PR per the title is to just add the toggle switch.

**Comment by Alirezaaraby** [comment]
> This looks good to me for this specific doc style change. I would consider swapping the accent colors to amber to match FastApi docs, but the scope of this PR per the title is to just add the toggle switch.

Firstly, typer doc has it's own style so i think there's no need to change the color to amber. And secondly, I wrote light/dark **mode** switch, and the mode is a key word here which shows this pr will add dark mode switch to typer docs :)

**Comment by svlandeg** [comment]
[maintenance note:] Related PR: https://github.com/tiangolo/typer/pull/418

**Review by svlandeg — COMMENTED** [review]
Hi @Alirezaaraby, thanks so much for the contribution!

It looks like in the meantime, dark mode for the docs has been introduced by Tiangolo in PR #908 🎉 As such, I think we can close this one, but thanks again! 🙏

**Pull request closed without merging**